review-of-international-studies.csl

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Review of International Studies</title>
+    <id>http://www.zotero.org/styles/review-of-international-studies</id>
+    <link href="http://www.zotero.org/styles/review-of-international-studies" rel="self"/>
+    <link href="http://www.zotero.org/styles/urban-studies" rel="template"/>
+    <link href="https://www.cambridge.org/core/services/aop-file-manager/file/57753670dd5679630ae895f8" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="political_science"/>
+    <issn>0260-2105</issn>
+    <eissn>1469-9044</eissn>
+    <updated>2016-12-17T15:27:43+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor-translator">
+    <names variable="editor translator" prefix="(" suffix=")" delimiter=", ">
+      <name and="symbol" initialize-with=". " delimiter=", "/>
+      <label form="short" prefix=", " text-case="capitalize-first"/>
+    </names>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name form="long"/>
+      <label form="short" text-case="capitalize-first" prefix=" (" suffix=")"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <choose>
```

Comment:
the substitute for author and author-short should almost certainly correspond. If you're putting in the container-title here but use the title or short-title in the text, how is a reader going to find this in the bibliography?